source/testing/testing-controllers.md

```diff
@@ -30,8 +30,8 @@ export default Ember.Controller.extend({
 });
 ```
 
-`setProps` sets a property on the controller and also calls a method. In our 
-generated test, ember-cli already uses the `moduleFor` helper to setup a test 
+The `setProps` action directly sets one property, and calls the method to set the other.
```

Comment:
I believe that in the context of the rest of the page, it makes more sense to say _the_, rather than _a_.

Before the code block, it says:
> Here we have a controller `PostsController` with two properties, a method that
> sets one of those properties, and an action named `setProps`.

Therefore, if you're reading top to bottom, I thought it was clearer that we're talking about that method, _the_ method. Rather than another method, _a_ method.

If you disagree, I'm happy to change it :)